vitest.config.ts

```diff
@@ -1,6 +1,7 @@
-import react from '@vitejs/plugin-react';
 import path from 'path';
 
+import react from '@vitejs/plugin-react';
+
```

Comment:
This was highlighted as an ESLint problem so thought I might as well fix it in one PR.